Makefile.include

```diff
@@ -747,6 +747,11 @@ test-input-hash:
 	true
 endif
 
+.PHONY: fuzz
+fuzz:
```

Comment:
```suggestion
fuzz: all-asan
```

So `make fuzz` will always be enough.

Reply:
This is theoretically possible, I didn't do this initially because I did not want to enforce usage of the address sanitizer. For instance, to allow instrumentation with other sanitizers (e.g. UBSAN). Besides, some bugs (memory leaks) can also be found without using any sanitizer at all as AFL itself detects a hang in these cases (e.g. #12001).